libpromises/evalfunction.c

```diff
@@ -1586,6 +1623,44 @@ static FnCallResult FnCallGetMetaTags(EvalContext *ctx, ARG_UNUSED const Policy
         tagset = EvalContextClassTags(ctx, ref.ns, ref.name);
         ClassRefDestroy(ref);
     }
+    else if (strcmp(fp->name, "getbundlemetatags") == 0)    
+    {
+        const char *bundleref = RlistScalarValue(finalargs);
+        const Rlist *args = RlistFromSplitString(bundleref, ':');
+        const char *namespace;
+        const char *name;
+        if (!args->next)
+        {
+            namespace = "default";
+            name = RlistScalarValue(args);
+        } else {
+            namespace = RlistScalarValue(args);
+            name = RlistScalarValue(args->next);
+        }
```

Comment:
```suggestion
        const char *namespace = (!args->next) ? "default" : RlistScalarValue(args);
        const char *name = (!args->next) ? RlistScalarValue(args) : RlistScalarValue(args->next);
```
You can consider using the ternary operator here (since we are not payed per line of code).

Reply:
Or even use it inside params;

```
const char *name = RlistScalarValue((args->next == NULL) ? args : args->next);
```

(Also, our code style guidelines say to explicitly compare with NULL).